### ISSUE TITLE
Can't start version `12.10.4`

### ISSUE THREAD

**Issue by lenovouser** [sameersbn/docker-gitlab#2145]
```
2020-05-16 11:54:17,559 INFO supervisord started with pid 692
2020-05-16 11:54:18,560 INFO spawned: 'gitaly' with pid 714
2020-05-16 11:54:18,561 INFO spawned: 'puma' with pid 715
2020-05-16 11:54:18,562 INFO spawned: 'gitlab-workhorse' with pid 716
2020-05-16 11:54:18,563 INFO spawned: 'sidekiq' with pid 717
2020-05-16 11:54:18,564 INFO spawned: 'mail_room' with pid 722
2020-05-16 11:54:18,565 INFO spawned: 'nginx' with pid 723
2020-05-16 11:54:18,566 INFO spawned: 'sshd' with pid 724
2020-05-16 11:54:18,567 INFO spawned: 'cron' with pid 725
2020-05-16 11:54:19,357 INFO exited: mail_room (exit status 1; not expected)
2020-05-16 11:54:19,565 INFO success: gitaly entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-05-16 11:54:19,565 INFO success: puma entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-05-16 11:54:19,566 INFO success: gitlab-workhorse entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-05-16 11:54:19,566 INFO success: sidekiq entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-05-16 11:54:19,566 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-05-16 11:54:19,566 INFO success: sshd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-05-16 11:54:20,568 INFO spawned: 'mail_room' with pid 834
2020-05-16 11:54:20,568 INFO success: cron entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2020-05-16 11:54:21,197 INFO exited: mail_room (exit status 1; not expected)
2020-05-16 11:54:23,200 INFO spawned: 'mail_room' with pid 835
2020-05-16 11:54:23,915 INFO exited: mail_room (exit status 1; not expected)
Missing Rails.application.secrets.openid_connect_signing_key for production environment. The secret will be generated and stored in config/secrets.yml.
rake aborted!
Psych::SyntaxError: (<unknown>): did not find expected key while parsing a block mapping at line 925 column 5
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/settingslogic-2.0.9/lib/settingslogic.rb:103:in `initialize'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/settingslogic-2.0.9/lib/settingslogic.rb:60:in `new'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/settingslogic-2.0.9/lib/settingslogic.rb:60:in `instance'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/settingslogic-2.0.9/lib/settingslogic.rb:37:in `[]'
/home/git/gitlab/config/initializers/1_settings.rb:6:in `<top (required)>'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2/lib/active_support/dependencies.rb:319:in `load'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2/lib/active_support/dependencies.rb:319:in `block in load'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2/lib/active_support/dependencies.rb:291:in `load_dependency'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2/lib/active_support/dependencies.rb:319:in `load'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/engine.rb:667:in `block in load_config_initializer'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2/lib/active_support/notifications.rb:182:in `instrument'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/engine.rb:666:in `load_config_initializer'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/engine.rb:624:in `block (2 levels) in <class:Engine>'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/engine.rb:623:in `each'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/engine.rb:623:in `block in <class:Engine>'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/initializable.rb:32:in `instance_exec'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/initializable.rb:32:in `run'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/initializable.rb:61:in `block in run_initializers'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/initializable.rb:50:in `each'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/initializable.rb:50:in `tsort_each_child'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/initializable.rb:60:in `run_initializers'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/application.rb:363:in `initialize!'
/home/git/gitlab/config/environment.rb:5:in `<top (required)>'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2/lib/active_support/dependencies.rb:325:in `require'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2/lib/active_support/dependencies.rb:325:in `block in require'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2/lib/active_support/dependencies.rb:291:in `load_dependency'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/activesupport-6.0.2/lib/active_support/dependencies.rb:325:in `require'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/application.rb:339:in `require_environment!'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/railties-6.0.2/lib/rails/application.rb:515:in `block in run_tasks_blocks'
/home/git/gitlab/vendor/bundle/ruby/2.6.0/gems/rake-12.3.3/exe/rake:27:in `<top (required)>'
Tasks: TOP => db:migrate => db:load_config => environment
(See full trace by running task with --trace)
```

Seems like it can't parse a `yaml` file, but I don't know which one it is?

**Comment by solidnerd** [comment]
Hey @lenovouser,
I think this happened through the latest sync of the upstream configs. I'll look into into it. My current assumption is that https://github.com/sameersbn/docker-gitlab/blob/master/assets/runtime/config/gitlabhq/gitlab.yml#L1084 this breaks the start.

**Comment by solidnerd** [comment]
I got it  @lenovouser  see my commit it will be available in 12.10.6. I`ll release this in the evening ~ 6pm GMT+2

**Comment by paolomainardi** [comment]
It worked perfectly, thanks again @solidnerd you rock!
